### PR TITLE
Wildlife

### PR DESCRIPTION
**Pull request by TeaTimeDino**
# Description

This is my plan to the beginning adding some more liveliness to the map. For now this includes a rabbit, beaver and opossum. If this goes well and people like it, I will work on adding more.
These are all passive mobs, merely there to be cute.

<details><summary><h1>Wildlife</h1></summary>
<p>

still don't know what goes here

</p>
</details>

---
<details><summary><h1>Media</h1></summary>
# Changelog

:cl:

### CODE REVIEW

File: Resources/Prototypes/_Nuclear14/Entities/Markers/Spawners/mobs.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Passive Animals
+#MARK: Passive Animals
```
Comment (Partmedia) [inline Resources/Prototypes/_Nuclear14/Entities/Markers/Spawners/mobs.yml:1]: What is the "MARK:" change on this and subsequent lines for?

Comment (TeaTimeDino) [reply Resources/Prototypes/_Nuclear14/Entities/Markers/Spawners/mobs.yml:1]: It just shows up on the side when you are scrolling through the yaml. It's to make things easier to find. Shouldn't change the code

Comment (Peptide90) [reply Resources/Prototypes/_Nuclear14/Entities/Markers/Spawners/mobs.yml:1]: It's something the yaml plugin does for VSCode I'm told.

File: Resources/Prototypes/_Nuclear14/Entities/Markers/Spawners/mobs.yml [hunk 2]
```diff
@@ -59,7 +59,82 @@
...
+      - state: green
+      - sprite: _Nuclear14/Mobs/Animals/rabbit.rsi
+        state: rabbit
+      - state: ai
+  - type: ConditionalSpawner
+    prototypes:
+      - N14MobRabbit
+
+- type: entity
+  name: Normal Opossum Spawner
```
[the remark this answers is not in the document]
Comment (TeaTimeDino) [reply Resources/Prototypes/_Nuclear14/Entities/Markers/Spawners/mobs.yml:108]: They should spawn different versions of the animal. I made a fallout version of them and ones that hadn't been affected by the fallout. But I could get wrong in how my yaml reflects that

[the remark this answers is not in the document]
Comment (Peptide90) [reply Resources/Prototypes/_Nuclear14/Entities/Markers/Spawners/mobs.yml:108]: Ah yes I see what it is. So the sprite component on the spawner is just for the actual spawn marker, and doesn't affect the mob that is spawned.

You'll need a normal version of each mob with the normal sprite, and a waster version with that sprite. You can use parenting on them to cut down duplication but I imagine the normal versions might have non irradiated meat.

[the remark this answers is not in the document]
Comment (TeaTimeDino) [reply Resources/Prototypes/_Nuclear14/Entities/Markers/Spawners/mobs.yml:108]: Ah I see, cool. Thanks!

**Pull request merged**